### DISCUSSION TITLE
Hybrid executors support in Airflow Helm Chart

### DISCUSSION THREAD

**Discussion by hadarsharon** [apache/airflow#42437] [Q&A]
Hi - I am aware that Hybrid Excecutors is currently an experimental feature, but I'd like to know if there is any plan to support it in the Airflow Helm Chart in the foreseeable future?

Thanks

**Comment by potiuk** [comment]
As soon as sometimes add it (could be you or anyone who wants it, it will be added. This is how community development works. Somoene who needs it, adds it and contributes for others. It might be you.

**Comment by hadarsharon** [comment]
@potiuk So that is to say, you are not aware of any existing or planned development on anyone's behalf as things stand. 

I am aware that I could work on it or add it, but working on the same thing twice (especially if someone's already ahead of you) is counterproductive, hence my original question :)

**Comment by potiuk** [comment]
> @potiuk So that is to say, you are not aware of any existing or planned development on anyone's behalf as things stand.

If no-one tells they are working on it, then no-one knows it ;).  Other than that Airflow is developed by ~3000 contributors, most of them individually contributing their work on theor own agenda and schedule. This is how open-source work. If you expect there will be single person that knows with certainty what is the roadmap and what every one of those contributors work on, then no - this is not how it works. 

If a PR is opened, then someone works on it - search through the PRs to find out if someone started to work on - it. You have the same mechanism to find it as anyone else here. Look at the devlist discussions, if you want to really be proactive - ask on the devlist, but you will never be 100% sure.

And yes it means that sometimes people work in parallel on the same thing - which is eventually a good thing, because then better solution can be chosen and even often those people who work on similar things realise they do, they band together, review each other's code and maybe even cross-polinate their solutions to make a common, better one. 

So no - it's not counterprodactive and inefficient as a "whole" for the product even if people start working on the same thing in parallel occasionally.
